.sample.env

```diff
@@ -0,0 +1,7 @@
+PORT=
```

Comment:
I need to learn about the .env - I used it for my GitHub fetcher heroku deployment, but haven't messed with it since.  Seems like a cleaner way vs using config.js